### PR TITLE
Deterministic Uglify

### PR DESCRIPTION
**Pull request by thomas-huston-zocdoc**
@ZocDoc/polaris-devs 

Changes
--
- The new version of uglify I updated us to was giving non-deterministic output, so given the same JS input it would minify variables differently from one run to the next. This adds a fix to make it deterministic, so our integration tests can be reliable.

### CODE REVIEW

File: test-runner/package.json [hunk 1]
```diff
@@ -2,5 +2,6 @@
     "description": "runs the tests in parallel for faster results",
     "dependencies": {
         "async": "~0.8.0"
-    }
+    },
+    "main": "parallel-test-runner.js"
```
Comment (thomas-huston-zocdoc) [inline test-runner/package.json]: This makes it so you can just say `node .\test-runner` instead of having to specify which js file in this folder to run.

**Comment by Andrew-Hagedorn-ZocDoc** [comment]
:boom: 

image

File: test-runner/package.json (hunk 1 above)
Comment (Andrew-Hagedorn-ZocDoc) [inline test-runner/package.json]: image

File: src/minify/minify-js.js [hunk 2]
```diff
@@ -35,6 +35,8 @@ function minify(options) {
 
 function generateSyntaxTree(js, filePath) {
 
+    uglify.base54.reset();
```
Comment (Andrew-Hagedorn-ZocDoc) [inline src/minify/minify-js.js]: Would this have caused our cachebusters to update day over day?

Comment (thomas-huston-zocdoc) [inline src/minify/minify-js.js]: It probably would, or at least could, have. This hasn't been merged into the monolith yet, though.

Comment (Andrew-Hagedorn-ZocDoc) [inline src/minify/minify-js.js]: Gotcha

**Pull request merged**